pac4j-jwt/src/main/java/org/pac4j/jwt/profile/JwtGenerator.java

```diff
@@ -114,9 +126,15 @@ protected String encryptJwt(final SignedJWT signedJWT) throws Exception {
     }
 
     protected SignedJWT signJwt(final JWTClaimsSet claims) throws JOSEException {
```

Comment:
No.. But I kept it for backwards compatibility since it's protected.. I can remove it if you'd like. Please confirm.